src/main/java/it/sweven/blockcovid/configurations/WebSecurityConfiguration.java

```diff
@@ -104,4 +122,12 @@ FilterRegistrationBean disableAutoRegistration(TokenAuthenticationFilter filter)
   AuthenticationEntryPoint forbiddenEntryPoint() {
     return new HttpStatusEntryPoint(FORBIDDEN);
   }
+
+  @Bean
+  public AuthenticationProvider daoAuthenticationProvider() {
```

Comment:
This bean can be safely removed, the authentication provider has been moved to our `TokenAuthenticationProvider` implementation